### PR TITLE
Anthropic Claude 3 support (Sonnet, Opus, Haiku, and Vision)

### PR DESCRIPTION
**Pull request by gogwilt**
* Adds Claude 3 models (Sonnet, Opus, Haiku)
* Uses Anthropic's Messages API for Claude 3 models.
* Adds Claude Vision support via images in messages.
* Updates anthropic.md documentation.

Also: makes it a BIT easier to develop on the node-executor

This was originally PR #369, but I don't have permissions on @wayne-chang's fork.

### CODE REVIEW

**Comment by gogwilt** [comment]
Some potential TODOs:
- [ ] System prompts - I need to test locally whether system prompts work as expected.
- [ ] Token counts - Anthropic's streaming API actually returns accurate token counts, so we can give more accurate cost estimates.

File: packages/core/bundle.esbuild.ts [hunk 1]
```diff
@@ -19,18 +19,26 @@ const aliasModule = (moduleFrom: string, moduleTo: string): esbuild.Plugin => ({
   },
 });
 
-esbuild.build({
+const options: esbuild.BuildOptions = {
```
Comment (gogwilt) [inline packages/core/bundle.esbuild.ts:22]: @abrenneke I'm not sure what I was doing here. Sourcemaps are still not working for me in VSC debugger, but I was able to use the JS files to debug OK.

Comment (abrenneke) [reply packages/core/bundle.esbuild.ts:22]: Yeah I had to add this, it's fine!

**Review by abrenneke — APPROVED** [review]
Looks awesome, thank you.

**Pull request merged**